src/ngrest/plugins/Injector.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace luya\admin\ngrest\plugins;
+
+use luya\admin\ngrest\base\Plugin;
+use luya\helpers\Html as HtmlHelper;
+
+/**
+ * Create a generic form input based on ZAA Directives.
+ *
+ * ```php
+ *   public function ngRestAttributeTypes()
+ *   {
+ *      return [
+ *          'name' => 'text',
+ *          'type' => 'text',
+ *          'value' => ['injector', 'directiveField' => 'type'],
+ *      ];
+ *   }
+ * ```
+ *
```

Comment:
So maybe we could also tell people when this is needed? Because this is a very specific use case, where you like to assign a certain type from a dynamic field input. 

Maybe it would also be more clear with an example like this:

```php
return [
         'type' => ['selectArrray', 'data' => [self::TYPE_PASSWORD => 'Passwort Input', self::TYPE_TEXT => 'Text Input']],
         'value' => ['injector', 'directiveField' => 'type'],
 ];
```